facade/src/main/java/com/redhat/lightblue/migrator/facade/DAOFacadeBase.java

```diff
@@ -208,6 +208,15 @@ public T call() throws Exception {
         }
     }
 
+    private Throwable extractUnderlyingException(Throwable t) {
```

Comment:
It's not perfect, because if the service returns ExecutionException or InvocationTargetException, facade will tamper with it. Do you think I should make it more strict?